src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,114 @@
+package core.basesyntax.service;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.dao.StorageDao;
+import core.basesyntax.dao.StorageDaoImpl;
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.User;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.Test;
+
+class RegistrationServiceImplTest {
+
+    private StorageDao storage = new StorageDaoImpl();
+    private RegistrationService registrationService = new RegistrationServiceImpl();
+
+    @AfterEach
+    void tearDown() {
+        Storage.people.clear();
+    }
+
+    @Test
+    void register_addToStorage_Ok() {
+        User firstUser = new User();
+        firstUser.setLogin("drapail");
+        User actual = storage.add(firstUser);
+        User expected = firstUser;
+        assertEquals(expected, actual);
+    }
+
+    @Test
+    void register_getExistingUserFromStorage_Ok() {
+        User firstUser = new User();
+        firstUser.setLogin("drapail");
+        storage.add(firstUser);
+        User actual = storage.get("drapail");
+        User expected = firstUser;
+        assertEquals(expected, actual);
+    }
+
+    @Test
+    public void register_duplicateLogin_notOk() {
+        User firstUser = new User();
+        User secondUser = new User();
+        firstUser.setLogin("drapail");
+        secondUser.setLogin("drapail");
```

Comment:
you should set all valid fields to user(age, password). Same in other methods. You should set all valid fields except one, that you testing